Userland/Libraries/LibJS/Runtime/Temporal/PlainDate.h

```diff
@@ -7,8 +7,10 @@
 
 #pragma once
 
+// FIXME
 #include <LibJS/Runtime/Completion.h>
 #include <LibJS/Runtime/Object.h>
+#include <LibJS/Runtime/Value.h>
```

Comment:
You want to add a header for Value (implicitly pulled in literally everywhere) but not for `GlobalObject`, `FunctionObject`, `String`, and `StringView` (all used in this header but not explicitly included)?

Reply:
To answer both of your comments: Originally, it didn't compile without the #include for Value.h, so I added it in. Then, one day when rebasing, I ran into a merge conflict, wanted to make it compile again so I accepted *all* the includes with a FIXME to remind myself to pare the includes down again. Well, apparently I forgot this one, sorry! :sweat_smile: